numpy/core/_add_newdocs.py

```diff
@@ -1056,8 +1056,7 @@
             decimal numbers, an operation which is better spelt
             ``frombuffer(string, dtype, count)``. If `string` contains unicode
             text, the binary mode of `fromstring` will first encode it into
-            bytes using either utf-8 (python 3) or the default encoding
-            (python 2), neither of which produce sane results.
```

Comment:
For deprecation explanations, I'd argue we should keep the references to old python versions. People upgrading their code want to know what it _used_ to do in their old numpy version, which may still support the older python versions.